### PR TITLE
Fix for double tap zoom for PDFs

### PR DESCRIPTION
**Pull request by sujaygarlanka**
### Issue Link :link:
- https://jira.inside-box.net/browse/SDK-1183

### Goals :soccer:
- Fix single tap full screen mode on PDFs
- Fix double tap zoom on PDFs

### Implementation Details :construction:
- Added custom gestures to `pdfView` and removed them from `view`

### Testing Details :mag:
- Manual testing

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @sujaygarlanka has signed the CLA. Thanks for the pull request!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 
-## In Progress
+## Next Release
 
 __Breaking Changes:__
 
 __New Features and Enhancements:__
 
 - Back button for search results exits out of search view, allowing the user to view the file instead of exiting to folder items table view 
+- Fixed single tap full screen mode on PDFs
```
Comment (PJSimon) [inline CHANGELOG.md]:
```suggestion
- Fixed single tap for full screen mode toggle on PDFs
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+- Fixed double tap zoom on PDFs
```
Comment (PJSimon) [inline CHANGELOG.md]:
```suggestion
- Fixed double tap zoom level on PDFs
```

File: Sources/Core/UI/Viewers/PDF/PDFViewController.swift [hunk 3]
```diff
@@ -567,6 +566,7 @@ extension PDFViewController: UIGestureRecognizerDelegate {
 
 extension PDFViewController {
     @objc func doubleTapped(_ sender: UITapGestureRecognizer) {
+        NSLog("test")
```
Comment (PJSimon) [inline Sources/Core/UI/Viewers/PDF/PDFViewController.swift]:
```suggestion
```

Comment (PJSimon) [reply Sources/Core/UI/Viewers/PDF/PDFViewController.swift]: Is this still needed or should it be removed?

**Pull request merged**